hive/agents/drqn.py

```diff
@@ -277,7 +316,20 @@ def update(self, update_info, agent_traj_state=None):
                 1 - batch["done"]
             )
 
-            loss = self._loss_fn(pred_qvals, q_targets).mean()
+            if self._burn_frames > 0:
+                interm_loss = self._loss_fn(pred_qvals, q_targets)
+                mask = torch.zeros(
+                    self._replay_buffer._max_seq_len,
+                    device=self._device,
+                    dtype=torch.float,
+                )
+                mask[self._burn_frames :] = 1.0
+                mask = mask.view(1, -1)
+                interm_loss *= mask
+                loss = interm_loss.mean()
```

Comment:
Isn't the correct way of doing it `interm_loss.sum() / mask.sum()`?

Reply:
That's true. This should be fixed after merging #339.